yarnitor/background/worker.py

```diff
@@ -305,6 +307,9 @@ def _generate_listing(self):
             return {}
         apps = cluster_apps['apps']['app']
 
+        # Number of apps with unknown state
+        unknown_state = 0
```

Comment:
The increment got lost in a rebase-patch. Good catch. (And I'll work on #1 soon!)